hr_recruitment_skill/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Employee Skills
```

Comment:
Could you please rename the "Employee Skills"--> "Job Skills" or "Applicant Skills" ?

Reply:
Hello, or maybe I should follow the technical name of the module with something like "HR recruitment skill'"

Reply:
"Recruitment Skill" is also good to me.